app/src/main/java/com/grzegorzdec/tagmango/model/Client.kt

```diff
@@ -0,0 +1,9 @@
+package com.grzegorzdec.tagmango.model
+
+data class Client(
+    val id: String = "",
+    val name: String = "",
+    val latitude: Double = 0.0,
```

Comment:
maybe default lat/lng should point someware in wroclaw? In case of any api problem map may try to center all points (including wroclaw and lat 0 long 0) and it may looks bad.

Reply:
Good point, will change it :)